src/SysInfo.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2023 - Ed Beroset <beroset@ieee.org>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "SysInfo.h"
+
+SysInfo::SysInfo(QObject *parent) :
+    QObject(parent)
+{
+    sysinfo(&info);
+    copy_loads();
+}
+
+void SysInfo::copy_loads() {
```

Comment:
Here too, to keep things consistent with the rest of the C++ methods this should be copyLoads

Reply:
OK